tdrs-frontend/src/actions/sttList.js

```diff
@@ -39,6 +39,7 @@ export const fetchSttList = () => async (dispatch) => {
     })
 
     if (data) {
+      // shouldn't this logic be done by the backend serializer?
```

Comment:
this may not be worthy of an entire issue, but when I was working through this I saw this code and it seems to me like like this is the sort of thing django should be handling before the data ever gets to the react UI.

I am open to thoughts on if/how we should deal with this.

Reply:
Is this supposed to be removing Federal Government from the options list? If so, I definitely agree it should be done by the backend based on the permissions of the person requesting. Also, it doesn't appear to work because I still see the option for "Federal Government" in both the STT combobox on the upload form and the Profile page.

I'd think a separate issue would be warranted to implement that correctly on the backend. The only people who should see that are OFA admins as far as I know.

Reply:
agree. i'd only expect to see fedgov on the frontend request access and backend profile comboboxes because this is the expected STT for any OFA staff (sys admin, ofa admin, ofa data analyst, regional). good to set this up so that these user groups cannot upload/download on behalf of that STT (e.g. when profile STT = fedgov, data files STT != fedgov)